### ISSUE TITLE
DB tables are missing constraints

### ISSUE THREAD

**Issue by karlmdavis** [karlmdavis/rps-tourney#106]
This is rather horrifying, and I'm glad I caught it now. Apparently, either my Liquibase syntax is wrong or Liquibase itself has a huge bug.

I've identified the following missing FKs:

* `AccountRoles.accountId`
* `AuthTokens.accountId`
* `GuestLoginIdentities.id`
* `Games.player1Id`
* `Games.player2Id`
* `Players.humanAccountId`
* `GameRounds.gameid`
* `LoginIdentities.accountId`
* `AuditAccountMerges.targetAccountId`
* `AuditAccountLoginMerges.auditAccountMergeId`
* `AuditAccountLoginMerges.loginidentityid`
* `AuditAccountGameMerges.parentAuditAccountMergeId`
* `AuditAccountGameMerges.gameid`

Basically, every single FK except for `GameLoginIdentities.id` is missing. Gah!!!!
